projects/angular/src/wizard/_properties.wizard.scss

```
@@ -25,7 +27,47 @@
       --clr-wizard-stepnav-active-border-radius: var(--clr-wizard-border-radius);
 
       --clr-wizard-stepnav-border-color: hsl(0, 0%, 89%);
-      --clr-wizard-box-shadow-color: rgba(0, 0, 0, 0.2);
+      --clr-wizard-box-shadow: 0
+        #{variables.$clr_baselineRem_1px}
+        #{variables.$clr_baselineRem_2px}
+        #{variables.$clr_baselineRem_2px}
+        rgba(0, 0, 0, 0.2);
```

Reintroduce `--clr-wizard-box-shadow-color: rgba(0, 0, 0, 0.2);` some app might overloading it.

```suggestion
        var(--clr-wizard-box-shadow-color);
```

---

Thanks for catching. I added `clr-wizard-box-shadow` with the idea to use the color too but missed it on the way.